app/views/fruits/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render 'boot'%>
+<div class="text-center py-2 bg-warning text-white text-capitalize">
```

Comment:
Forgot to include this in the review: another color contrast issue.  White on bg-warning has a ratio of 1.63:1, rather than the required 3:1.